src/django_elasticsearch_dsl_drf/filter_backends/suggester/functional.py

```diff
@@ -67,7 +67,7 @@
     >>>
     >>>         model = Publisher  # The model associate with this Document
 """
-from elasticsearch_dsl.search import AggsProxy
```

Comment:
@miguelgrinberg I started implementing a solution in #319 .

However AggsProxy is used as well [in another class](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/21a2a61c4622db92240dfeaef84f67578060305d/src/django_elasticsearch_dsl_drf/filter_backends/suggester/functional.py#L399). I am not sur how to tackle this one...

Reply:
@millerf I have missed that second usage. Unfortunately that one is more difficult, because it uses a few other private items in addtion to the `AggsProxy` instance. The whole idea of updating a `Search` instance in place goes against our design.

The two ways I can think of to "clean" one of these objects are:
- Create a brand new instance of the same class
- Run a query that is guaranteed to return no results

Reply:
> @millerf I have missed that second usage. Unfortunately that one is more difficult, because it uses a few other private items in addtion to the `AggsProxy` instance. The whole idea of updating a `Search` instance in place goes against our design.
> 
> The two ways I can think of to "clean" one of these objects are:
> 
> * Create a brand new instance of the same class
> * Run a query that is guaranteed to return no results

Unfortunately I don't know enough about this plugin to get to do that... I am hoping someone will be able to help...